src/test/java/io/vertx/spi/cluster/zookeeper/MockZKCluster.java

```diff
@@ -64,27 +64,14 @@ public void stop() {
   public ClusterManager getClusterManager() {
     if (server == null) {
       try {
-        server = new TestingServer(new InstanceSpec(null, -1, -1, -1, true, -1, 10000, 120), true);
+        server = new TestingServer(spec, true);
       } catch (Exception e) {
         e.printStackTrace();
       }
     }
-    String connectString = server.getConnectString();
-    curator = CuratorFrameworkFactory.builder()
-        .namespace("io.vertx")
-        .sessionTimeoutMs(3000)
-        .connectionTimeoutMs(2000)
-        .connectString(connectString)
-        .retryPolicy(retryPolicy).build();
-      curator.start();
-      try {
-        curator.blockUntilConnected(2, TimeUnit.SECONDS);
-      } catch (Exception e) {
-          e.printStackTrace();
-      }
-      String uuid = UUID.randomUUID().toString();
-      ZookeeperClusterManager zookeeperClusterManager = new ZookeeperClusterManager(curator, uuid);
-      clusterManagers.add(zookeeperClusterManager);
-      return zookeeperClusterManager;
+
+    ZookeeperClusterManager zookeeperClusterManager = new ZookeeperClusterManager(getDefaultConfig());
```

Comment:
Some of the tests that were failing for me locally, seemed to fail against the Test ZooKeeper server, but when I ran it against a locally configured server they were fine, using the defaults of `ZookeeperClusterManager`. Although this change hasn't made much help when running in CI, I seems still simpler and better to just use the defaults that we are defining in `ZookeeperClusterManager` + config JsonObject.